docs/UserGuide.md

```diff
@@ -1,186 +1,239 @@
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
+3. Double-click on the file to start the application. The GUI should appear in a few seconds.<br>
+> <img src="images/startup_screen.jpg" width="600">
 
+4. Get started by adding your first task that you have in mind! Refer to the "2. Features" for further instructions. <br>
+> <img src="images/add_screen.jpg" width="600">
+
+
+5. After step 6, you can view your task in 3 different tabs.
+
+   > **Today/Date**
+   >
+   > * Shows you the list of tasks you have for the specified date. By default, it will show the tasks for today. Click left/right arrow to navigate between the dates.
+   > <img src="images/today_screen.jpg" width="600">
+   >
+   > **Overview**
+   >
+   > * Shows you an overall view of the dates. The highest priority tasks will be shown as a preview for every single date. Scroll to navigate between the dates.
+   > <img src="images/overview_screen.jpg" width="600">
+   >
+   > **Priority**
+   >
+   > * Shows you 4 options of priority tabs, from highest to lowest, which are indicated by different colours. Selecting the tab allows you to view the list of tasks that is under that priority. Eg. By clicking on â€œHighest Priorityâ€� tab, it will bring you to the list of tasks that you have in it.<br>
+   > <img src="images/priority_screen.jpg" width="600">
 
 ## 2. Features
 
-> **Command Format**
+> Command Format
 >
 > * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
+> * Items in `SQUARE_BREACKETS` are optional.
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
 ### 2.1. Viewing help : `help`
+Format: `help`<br>
 
-Format: `help`
-
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> Help is shown if you enter an incorrect command e.g. `asdf` <br>
+> Alternatively, type /help to obtain a list of commands that you can use.
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a Task : `add`
+Adds a task to the task manager. <br>
+Format: `add TASK_NAME d/DEADLINE p/PRIORITY_LEVEL [i/ANY_INFO] [t/TAGS]...`
+   > `DEADLINE` will be in the format of "date month (indicated by first three letters) year"
+   > `PRIORITY_LEVEL` should be defined by integers 1 (high) to 4(low).<br>
+   > `ANY_INFO` allows you to key in details with regards to the task, optional.<br>
+   > `TAGS` allows you to assign tags to the tasks, optional.
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Examples:
+* add CS3230 Assignment 1 d/8 Jan 2018 p/1 i/How to do? t/School t/CS3230
+* add Buy Milk d/23 Feb 2017 p/4 t/Chores
```

Comment:
How about floating tasks and timed tasks (with a from and to)?  You have to explicitly support these three types of tasks.